python/lsst/meas/deblender/baseline.py

```diff
@@ -723,12 +724,12 @@ def newDeblend(debPlugins, footprint, mMaskedImage, psfs, psfFwhms,
     # Import C++ routines
 
     if log is None:
-        import lsst.log as lsstLog
-
-        log = lsstLog.Log.getLogger(__name__)
+        log = lsst.utils.logging.getLogger(__name__)
 
         if verbose:
```

Comment:
I retained the logic where `verbose` is only used if `log is None`. That might not be intentional but it is what was implemented before. The logger is reset if verbose is not set because (1) by default it's WARNING and (2) otherwise calling this with verbose True and then False would leave the trace logger set.